apps/server/src/modules/user/service/user.service.spec.ts

```diff
@@ -385,4 +386,28 @@ describe('UserService', () => {
 			});
 		});
 	});
+
+	describe('findUsers is called', () => {
+		it('should call the repo with given query and options', async () => {
+			const query: UserQuery = {
+				schoolId: 'schoolId',
+				isOutdated: true,
```

Comment:
though this is a legitimate test for the function, you might want to test the newly added scope, too (see next comment, which one it should be ;))